engine/engine_test.go

```diff
@@ -555,6 +557,35 @@ func TestBaseOutput(t *testing.T) {
 	assert.Equal(t, "bar", output["foo"])
 }
 
+func TestScriptPlugin(t *testing.T) {
+	res, err := createResolution("execScript.yaml", nil, nil)
+	assert.NotNil(t, res)
+	assert.Nil(t, err)
+
+	res, err = runResolution(res)
+	assert.NotNil(t, res)
+	assert.Nil(t, err)
+
+	payload := make(map[string]interface{})
+	payload["msg"] = "Hello world!"
```

Comment:
one silly suggestion: make `world` an input to the template, 
instead of making the argv a fixed feature in the template execScript.yaml
defining that value in the test would make it more readable and robust, imho